### ISSUE TITLE
matthewgraybosch.com no longer qualifies

### ISSUE THREAD

**Issue by matthewgraybosch** [kevquirk/512kb.club#240]
Please remove matthewgraybosch.com at your convenience. I've gone back to using WordPress and gotten happy with embeds so my site no longer meets the 512kb.club membership requirements.

**Comment by kevquirk** [comment]
Noooo! Always sad to see a website leave. You know though, you can get a WordPress site way below 512KB. For example, [my site](http://kevq.uk) is WordPress and comes in at ~225KB.

Sounds like a good challenge to get it back in the club! :-)

For now, I've removed your site. Thanks for letting me know.

**Comment by matthewgraybosch** [comment]
On Mon, 01 Mar 2021 00:24:53 -0800
Kev Quirk <notifications@github.com> wrote:

> Noooo! Always sad to see a website leave. You know though, you can
> get a WordPress site way below 512KB. For example, [my
> site](http://kevq.uk) is WordPress and comes in at ~225KB.
> 
> Sounds like a good challenge to get it back in the club! :-)
> 
> For now, I've removed your site. Thanks for letting me know.

Thanks for the understanding and encouragement. I've actually been
reading your site and I got the idea for providing a guestbook instead
of comments from you. I've found that the problem is that I want my
front page to show full posts, and that I like embedding multimedia
better than linking to it. I was able to mitigate some of the size
downside with lazy-loading and gotten the front page down to 800KB
uncompressed, but I still have more work to do. :)

-- 
Matthew Graybosch
https://matthewgraybosch.com
"Even in the future nothing works."
